mars/data/mars_datamanager.py

```diff
@@ -84,7 +88,14 @@ def next_eval_image(self, step: int) -> Tuple[int, RayBundle, Dict]:
             assert camera_ray_bundle.camera_indices is not None
             image_idx = int(camera_ray_bundle.camera_indices[0, 0, 0])
             object_rays_info = self.eval_dataset.metadata["obj_info"][image_idx]
-            camera_ray_bundle.metadata["object_rays_info"] = object_rays_info.reshape(
+
+            # PR: Expand tensor here
+            image_width = self.eval_dataset.cameras[image_idx].width
+            image_height = self.eval_dataset.cameras[image_idx].height
+
+            object_rays_info_view = object_rays_info.expand(image_height[0], image_width[0], -1, -1)
```

Comment:
you can use `camera_ray_bundle.shape[0], camera_ray_bundle.shape[1]` instead of `image_width` and `image_height`?

Reply:
Hi @Nplace-su,

I removed the comments and the deprecated code. I removed permute as well, as is not necessary anymore. Also, I now use `camera_ray_bundle.shape` in `mars_pipeline.py` and in the `mars_datamanager.py` eval method. 

AFAIK this change should be compatible with the current version of `cicai_render.py`. I tested with the PandaSet data parser, but testing on this change and on KITTI is highly welcome. Compatibility should be there as the "new" format can easily be broadcasted, thus not requiring a change in the `cicai_render.py`

Reply:
@j-pens LGTM! Great job!
@xBeho1der @wuzirui Could you plz take a look and merge this?